test/api/users.coffee

```diff
@@ -0,0 +1,85 @@
+'use strict'
+
+User = require("../../website/src/models/user").model
+app = require("../../website/src/server")
+
+describe "Users", ->
+
+  describe "Deleting Users", ->
+
+    before (done) ->
```

Comment:
I'd actually turn this into a beforeEach. Then you can just use a different user for each test, and make the test simpler.